haplotypecaller-gvcf-gatk3.b37.wgs.inputs.json

```diff
@@ -14,24 +14,34 @@
 
   "##_COMMENT4": "MISC PARAMETERS", 
   "HaplotypeCallerGvcf_GATK3.MergeVCFs.compression_level": 5,
+  "#HaplotypeCallerGvcf_GATK3.make_gvcf": "Boolean? (optional)",
+  "#HaplotypeCallerGvcf_GATK3.HaplotypeCaller.max_alt_alleles": "Int? (optional)",
+  "#HaplotypeCallerGvcf_GATK3.HaplotypeCaller.contamination": "Int? (optional)",
 
-  "##_COMMENT5": "DOCKERS (using gotc for picard now but will switch to picard-only docker when available)", 
-  "HaplotypeCallerGvcf_GATK3.picard_docker": "broadinstitute/genomes-in-the-cloud:2.3.0-1501082129",
-  "HaplotypeCallerGvcf_GATK3.gatk_docker": "broadinstitute/gatk3:3.8-0",
+  "##_COMMENT5": "DOCKERS - uncomment the following line and supply a valid docker container to override the default", 
+  "#HaplotypeCallerGvcf_GATK3.picard_docker_override": "String? (optional)",
+  "#HaplotypeCallerGvcf_GATK3.gatk_docker_override": "String? (optional)",
   
-  "##_COMMENT6": "PATHS", 
-  "HaplotypeCallerGvcf_GATK3.HaplotypeCaller.gatk_path": "/usr/",
-  "HaplotypeCallerGvcf_GATK3.MergeVCFs.picard_path": "/usr/gitc/",
+  "##_COMMENT6": "PATHS - uncomment the following line and supply a valid path excutable to override the default", 
+  "#HaplotypeCallerGvcf_GATK3.picard_path_override": "String? (optional)",
+  "#HaplotypeCallerGvcf_GATK3.gatk_path_override": "String? (optional)",
 
   "##_COMMENT7": "JAVA OPTIONS", 
   "HaplotypeCallerGvcf_GATK3.HaplotypeCaller.java_opt": "-Xms8000m",
   "HaplotypeCallerGvcf_GATK3.MergeVCFs.java_opt": "-Xms2000m",
 
   "##_COMMENT8": "MEMORY ALLOCATION", 
-  "HaplotypeCallerGvcf_GATK3.HaplotypeCaller.mem_size": "10 GB",
-  "HaplotypeCallerGvcf_GATK3.MergeVCFs.mem_size": "3 GB",
+  "#HaplotypeCallerGvcf_GATK3.MergeVCFs.machine_mem_gb": "Int? (optional)",
```

Comment:
Logically I prefer this the original way since HaplotypeCaller gets called first.

Reply:
No, problem. I'll correct the order throughout the json.